app/renderer/js/main.js

```diff
@@ -410,7 +410,7 @@ class ServerManagerView {
 		// Open all the tabs in background, also activate the tab based on the index
 		this.activateTab(index);
 		// Save last active tab
```

Comment:
A comment explaining why this was required in another process would be good ! LGTM.

Reply:
Good idea, thanks. We can remove the comment once the db problems are fixed.